### PR TITLE
Fix test versions

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: deploy/shakenfist_ci/tests/test_floating_ips.py [hunk 1]
```diff
@@ -65,20 +60,27 @@ def test_simple(self):
...
-                self.assertEqual(200, r.status_code)
-                self.assertEqual('Floating IPs work!\n', r.text)
-                return
-
-            except Exception as e:
-                if attempts < 5:
-                    pass
-                else:
-                    raise e
+                success = 200 == r.status_code
```
Comment (mandoonandy) [inline deploy/shakenfist_ci/tests/test_floating_ips.py:74]: So you left this here to see if I'm reviewing?

**Review by mandoonandy — CHANGES_REQUESTED** [review]
One tiny python issue

File: deploy/shakenfist_ci/tests/test_floating_ips.py (hunk 1 above)
Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:74]: No, that's deliberate. It stores the value of the comparison of 200 and r.status_code in a variable called success, which is checked in the next line. Or am I missing something?

Comment (mandoonandy) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:74]: Would it be simpler to write:
```
if r.status_code == 200 and r.text.find('Floating IPs work!') != -1:
    return
```

Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:74]: He's a witch!

Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py]: Also yes.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**